### PR TITLE
Add dh-prod-trino application definition

### PR DESCRIPTION
**Pull request by rimolive**
## This Pull Request implements

Add `dh-prod-trino` project to deploy on prod ocp

## If migrating an Application to ArgoCD
- [X] I have completed the list of action items on [this page](https://aicoe.github.io/aicoe-cd/get_argocd_to_manage_your_app/)

### CODE REVIEW

File: manifests/overlays/prod/applications/data_hub/dh-prod-trino.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+metadata:
+  name: dh-prod-trino
+spec:
+  destination:
+    namespace: dh-prod-trino
+    server: https://api.ocp4.prod.psi.redhat.com:6443
+  project: data-hub
+  source:
+    path: trino/overlays/prod
+    repoURL: https://github.com/AICoE/idh-manifests.git
```
Comment (anishasthana) [inline manifests/overlays/prod/applications/data_hub/dh-prod-trino.yaml:12]: https://github.com/AICoE/idh-manifests/tree/production/trino does not have `trino/overlays/prod` as an available path

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: manifests/overlays/prod/applications/data_hub/dh-prod-trino.yaml (hunk 1 above)
Comment (rimolive) [reply manifests/overlays/prod/applications/data_hub/dh-prod-trino.yaml:12]: https://github.com/AICoE/idh-manifests/pull/132 is the PR to add trino prod overlay

**Review by anishasthana — COMMENTED** [review]
/lgtm

**Review by accorvin — COMMENTED** [review]
/approve

**Pull request merged**